README.md

```diff
@@ -67,6 +67,7 @@ Options:
 - `--conversation` or `-c`: Name of the conversation key
 - `--path` or `-p`: Path where the bot and the conversation directory are located (default: "./.jambots or it it doesn't exist ~/.jambots")
 - `--last` or `-l`: Continue with the last conversation created
+- `--no_pretty` or `-n`: Disables pretty formatting for the output
```

Comment:
What about `--format FORMAT` to select how to render the thing?
```
exe/jambots chat # Uses CLI
exe/jambots chat --format plain # no pretty
exe/jambots chat --format json # Some kind of json thing
```

And `--no-format`, `-n` as shortcut of `--format plain`

The idea is having an interface that is not likely to change in the future

Reply:
It's a good idea. I'll create an issue to change it https://github.com/artero/jambots/issues/17